CHANGELOG.md

```diff
@@ -30,6 +32,7 @@
 - Fixed customer groups prices ranges on configurable products
 - Fixed a bug where categories highlighting didn't work as expected on PLP powered by InstantSearch
 - Fixed a bug where excluded websites weren't taken into account while indexing customer prices on products. (thanks @kamilszewczyk)
+- Fixed a bug where full page cache (FPC) didn't work on category pages
```

Comment:
Should probably add [this note](https://github.com/algolia/algoliasearch-magento-2/pull/1725/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR36) as well

Reply:
Yeah that makes sense to me.